3.3/.sti/bin/assemble

```diff
@@ -1,23 +1,66 @@
 #!/bin/bash
 
-# For SCL enablement
-source .bashrc
+# This script uses these environment variables:
+#
+# DISABLE_COLLECTSTATIC: if not empty, inhibits execution of 'manage.py collectstatic'.
 
-set -e
+function assemble() {
+  # For SCL enablement
+  source .bashrc
```

Comment:
you want to have this globally enabled (ideally we should never be doing this), also sourcing files in functions is bad practice because it makes it hard to test, etc... also if you call this function twice, you will enable SCL twice... 

personally I would like to break the assemble function into smaller functions and not call it 'assemble'.